<!-- -->
<launch>

<node pkg="tf" type="static_transform_publisher" name="frame_broadcaster_test" args="-0.86 0.71 0.35 3.1415 0 0 base_link frame 100"/>

<!--read waypoints and publish next waypoint-->
	<node pkg="goal_manager" type="frame_goal_manager" name="frame_goal_mangager" output="screen">
		<param name="next_waypoint" value="/goal_point"/>
		<param name="waypoint_list" value="$(find goal_manager)/waypoints/x_axis_scan_fast.txt"/>
		<param name="distance" value="/frame_goal_distance"/>
		<param name="min_distance" value="0.01"/>
		<param name="path_publisher" value="/frame_path"/>
		<!--as lower frequency as longer the robot will wait at a reached goalpoint-->
		<param name="frequency" value="2"/>
		<param name="user_goal_input" value="/move_base_simple/goal"/>
		<param name="rabbit_frame" value="/frame_goal"/>
		<param name="robot_frame" value="/frame"/>
		<param name="fixed_frame" value="/base_link"/>
		<param name="waittime" value="0.1"/>
		<param name="wait_for_topic" value="true"/>
		<param name="wait_topic_name" value="/movement_finished"/>
		
	</node>
</launch>
